stdlib/public/runtime/Casting.cpp

```diff
@@ -100,6 +100,7 @@ static void _buildNameForMetadata(const Metadata *type,
 
   Demangle::DemangleOptions options;
   options.QualifyEntities = qualified;
+  options.ShowLocalDiscriminators = false;
```

Comment:
I think this one probably goes with `qualified`, like `ShowPrivateDiscriminators`. Because…